src/ControlSystem/ControlErrors/Size/Signal.hpp

```diff
@@ -0,0 +1,129 @@
+ * \param excision_boundary a Strahlkorper representing the excision
+ *        boundary in frame Frame.  Note that the excision boundary is assumed
+ *        to be a sphere in the grid frame.
+ * \param grid_frame_excision_boundary_radius the radius of the (assumed
+ *        spherical) excision boundary in the grid frame.
+ * \param time_deriv_apparent_horizon Time derivative of apparent_horizon,
+ *        where the time derivative is taken in frame Frame.  That is, this
+ *        should simply be lim dt->0
+ *        (apparent_horizon(t+dt)-apparent_horizon(t))/dt with no advective
+ *        terms.
+ * \param lapse_on_excision_boundary Lapse on the excision boundary.
+ * \param frame_components_of_grid_shift The quantity
+ *        \f$\beta^i \frac{\partial x^\hat{i}}{\partial x_i}\f$ (see below)
+ *        evaluated on the excision boundary.  This is a tensor in frame
+ *        Frame.
+ * \param spatial_metric_on_excision_boundary metric in frame Frame.
+ * \param inverse_spatial_metric_on_excision_boundary metric in frame Frame.
+ * \param function_of_time FunctionOfTime that is being controlled.
+ *        This function_of_time contains DataVectors of size 1.
+ *
+ * The characteristic speed that is needed here is
+ * \f{align}
+ *     v &= -\alpha -n_i\beta^i \\
+ *     v &= -\alpha -n_\hat{i}\hat{\beta}^\hat{i}
+ *           - n_\hat{i}\frac{\partial x^\hat{i}}{\partial t} \\
+ *     v &= -\alpha -n_\bar{i}\bar{\beta}^\bar{i}
+ *           - n_\bar{i}\frac{\partial x^\bar{i}}{\partial t} \\
+ *     v &= -\alpha - n_\hat{i}
+ *          \frac{\partial x^\hat{i}}{\partial x^i} \beta^i,
+ *  \f}
+ *  where we have written many equivalent forms in terms of quantities
+ *  defined in different frames.
+ *
+ *  Here \f$\alpha\f$ is the lapse, which is invariant under frame
+ *  transformations, \f$\n_i\f$, \f$\n_\hat{i}\f$, and \f$\n_\bar{i}\f$
+ *  are the metric-normalized normal one form to the Strahlkorper in the
+ *  grid, distorted, and inertial frames, and
+ *  \f$\beta^i\f$, \f$\hat{\beta}^\hat{i}\f$, and \f$\bar{\beta}^\bar{i}\f$
+ *  are the shift in the grid, distorted, and inertial frames.
+ *
+ *  Note that we decorate the shift with hats and bars in addition to
+ *  decorating its index, because the shift transforms in a non-obvious
+ *  way under frame transformations so it is easy to make mistakes.
+ *  To be clear, these different shifts are defined by
+ * \f{align}
+ *   \beta^i &= \alpha^2 g^{0i},\\
+ *   \hat{\beta}^\hat{i} &= \alpha^2 g^{\hat{0}\hat{i}},\\
+ *   \bar{\beta}^\bar{i} &= \alpha^2 g^{\bar{0}\bar{i}},
+ * \f}
+ *  where \f$g^{ab}\f$ is the spacetime metric, and they transform like
+ * \f{align}
+ * \hat{\beta}^\hat{i} &= \beta^i \frac{\partial x^\hat{i}}{\partial x^i}-
+ *  \frac{\partial x^\hat{i}}{\partial t}.
+ * \f}
+ *
+ * The quantity we pass as frame_components_of_grid_shift is
+ * \f{align}
+ * \beta^i \frac{\partial x^\hat{i}}{\partial x^i} & \\
```

Comment:
You probably don't want this `& \\`, but if it's an intentional stylistic choice that's fine.